src/app.js

```diff
@@ -708,7 +708,7 @@ var run = function() {
 			cb(files[fileNameFromKey(SOL_CACHE_FILE)]);
 			return;
 		}
-		var importRegex = /import\s*[\'\"]([^\'\"]+)[\'\"];/g;
+		var importRegex = /^\s*import\s*[\'\"]([^\'\"]+)[\'\"];/g;
```

Comment:
Currently, if someone comments out an import, browser-solidity will still parse them (and return an error message if they're missing). This update only matches on import statements that have nothing (other than spaces) in front of them.